galaxy_ng/app/settings.py

```diff
@@ -190,3 +191,96 @@
             "max_age": None,
         },
     }
+
+# Obtain values for Social Auth
+SOCIAL_AUTH_KEYCLOAK_KEY = os.getenv('SOCIAL_AUTH_KEYCLOAK_KEY', default=None)
+SOCIAL_AUTH_KEYCLOAK_SECRET = os.getenv('SOCIAL_AUTH_KEYCLOAK_SECRET', default=None)
+SOCIAL_AUTH_KEYCLOAK_PUBLIC_KEY = os.getenv('SOCIAL_AUTH_KEYCLOAK_PUBLIC_KEY', default=None)
+KEYCLOAK_PROTOCOL = os.getenv('KEYCLOAK_PROTOCOL', default=None)
+KEYCLOAK_HOST = os.getenv('KEYCLOAK_HOST', default=None)
+KEYCLOAK_PORT = os.getenv('KEYCLOAK_PORT', default=None)
+KEYCLOAK_REALM = os.getenv('KEYCLOAK_REALM', default=None)
+
+# Add settings if Social Auth values are provided
+if (SOCIAL_AUTH_KEYCLOAK_KEY and SOCIAL_AUTH_KEYCLOAK_SECRET
+        and SOCIAL_AUTH_KEYCLOAK_PUBLIC_KEY and KEYCLOAK_HOST
+        and KEYCLOAK_PORT and KEYCLOAK_REALM):
+    KEYCLOAK_ADMIN_ROLE = os.getenv('KEYCLOAK_ADMIN_ROLE', default='hubadmin')
```

Comment:
ok, ignore my first comment about using `PULP_` prefix... I see that you need to read from env here so it is possible to perform this if condition.

Reply:
👍